kedify-agent/templates/agent-deployment.yaml

```diff
@@ -52,8 +52,6 @@ spec:
                 secretKeyRef:
                   name: kedify-agent
                   key: apikey
-            - name: CLOUD_PROVIDER
-              value: {{ .Values.agent.billingCloudProvider }}
```

Comment:
hmm, if you remove it here, how this is propagated to agent?

Reply:
ah ok, the `cloudAccountID` is being passed [here](https://github.com/kedify/charts/blob/97cc622001a3deefb8b3f7006e03b2d9e10eec60/kedify-agent/templates/cr-install/cr-configmap.yaml#L20-L22)